CONTRIBUTING.md

```diff
@@ -141,13 +140,13 @@ You can either manually set the environment variables or use the `pytest.ini.exa
 
 Execute the following command to run the IoT Hub integration tests:
 
-`pytest azext_iot/tests/test_iot_ext_int.py`
+`pytest azext_iot/tests/iothub/test_iot_ext_int.py`
```

Comment:
After the last set of PR's we significantly re-organized the tests. To only ruin iothub integration tests...something like this should work

`pytest azext_iot/tests/iothub/ -k "_int"`

Reply:
And to only run unit tests `pytest azext_iot/tests/iothub/ -k "_unit"`